cg/meta/transfer/external_data.py

```diff
@@ -71,6 +73,9 @@ def transfer_sample_files_from_source(self, dry_run: bool, ticket_id: int) -> No
             source_path=self.get_source_path(customer=cust, ticket_id=ticket_id),
             destination_path=self.get_destination_path(customer=cust),
         )
+        if nanopore:
```

Comment:
I think this will result in an error if the nanopore folder is not created.
```suggestion
        destination_folder: Path = Path(self.destination_path % cust)
        if nanopore:
            destination_folder = destination_folder / "nanopore"
        Path(destination_folder).mkdir(exist_ok=True, parents=True)
```